modules/t/test-genome-DBs/multi/compara/method_link_species_set_tag.txt

```diff
@@ -1533,29 +1533,29 @@
 596	non_reference_species	pelodiscus_sinensis
 596	non_ref_coding_coverage	21308991
 596	non_ref_coding_length	28388868
-596	non_ref_dna_collection	{"chunk_size" => 10100000,"group_set_size" => 10100000,"masking_options" => "{default_soft_masking => 1}","overlap" => 100000}
+596	non_ref_dna_collection	{"chunk_size" => 10100000,"group_set_size" => 10100000,"masking" => "soft","overlap" => 100000}
 596	non_ref_genome_coverage	99748679
 596	non_ref_genome_length	2202483752
 596	num_blocks	721612
 596	param	T=1 L=3000 H=2200 O=400 E=30 --ambiguous=iupac
 596	reference_species	anolis_carolinensis
 596	ref_coding_coverage	23688760
 596	ref_coding_length	27370256
-596	ref_dna_collection	{"chunk_size" => 10000000,"masking_options" => "{default_soft_masking => 1}","overlap" => 0}
+596	ref_dna_collection	{"chunk_size" => 10000000,"masking" => "soft","overlap" => 0}
 596	ref_genome_coverage	196655651
 596	ref_genome_length	1799143587
-384	non_ref_dna_collection	{"chunk_size" => 10100000,"collection_name" => "sus_scrofa all","genome_db_id" => 132,"genome_name_assembly" => "sus_scrofa:Sscrofa10.2","group_set_size" => 10100000,"masking_options" => "{default_soft_masking => 1}","overlap" => 100000}
+384	non_ref_dna_collection	{"chunk_size" => 10100000,"collection_name" => "sus_scrofa all","genome_db_id" => 132,"genome_name_assembly" => "sus_scrofa:Sscrofa10.2","group_set_size" => 10100000,"masking" => "soft","overlap" => 100000}
 384	param	T=1 L=3000 H=2200 O=400 E=30 --ambiguous=iupac
-384	ref_dna_collection	{"chunk_size" => 30000000,"collection_name" => "homo_sapiens mammal","genome_db_id" => 90,"genome_name_assembly" => "homo_sapiens:GRCh37","include_non_reference" => 1,"masking_options" => "{default_soft_masking => 1}","masking_options_file" => "$ENSEMBL_CVS_ROOT_DIR/ensembl-compara/scripts/pipeline/human36.spec","overlap" => 0,"region" => "chromosome:HG1292_PATCH,chromosome:HG1287_PATCH,chromosome:HG1293_PATCH,chromosome:HG1322_PATCH,chromosome:HG1304_PATCH,chromosome:HG1308_PATCH,chromosome:HG962_PATCH,chromosome:HG871_PATCH,chromosome:HG1211_PATCH,chromosome:HG271_PATCH,chromosome:HSCHR3_1_CTG1"}
-390	non_ref_dna_collection	{"chunk_size" => 10100000,"collection_name" => "sus_scrofa all","genome_db_id" => 132,"genome_name_assembly" => "sus_scrofa:Sscrofa10.2","group_set_size" => 10100000,"masking_options" => "{default_soft_masking => 1}","overlap" => 100000}
+384	ref_dna_collection	{"chunk_size" => 30000000,"collection_name" => "homo_sapiens mammal","genome_db_id" => 90,"genome_name_assembly" => "homo_sapiens:GRCh37","include_non_reference" => 1,"masking" => "soft","masking_options_file" => "$ENSEMBL_CVS_ROOT_DIR/ensembl-compara/scripts/pipeline/human36.spec","overlap" => 0,"region" => "chromosome:HG1292_PATCH,chromosome:HG1287_PATCH,chromosome:HG1293_PATCH,chromosome:HG1322_PATCH,chromosome:HG1304_PATCH,chromosome:HG1308_PATCH,chromosome:HG962_PATCH,chromosome:HG871_PATCH,chromosome:HG1211_PATCH,chromosome:HG271_PATCH,chromosome:HSCHR3_1_CTG1"}
```

Comment:
```suggestion
384	ref_dna_collection	{"chunk_size" => 30000000,"collection_name" => "homo_sapiens mammal","genome_db_id" => 90,"genome_name_assembly" => "homo_sapiens:GRCh37","include_non_reference" => 1,"masking" => "soft","overlap" => 0,"region" => "chromosome:HG1292_PATCH,chromosome:HG1287_PATCH,chromosome:HG1293_PATCH,chromosome:HG1322_PATCH,chromosome:HG1304_PATCH,chromosome:HG1308_PATCH,chromosome:HG962_PATCH,chromosome:HG871_PATCH,chromosome:HG1211_PATCH,chromosome:HG271_PATCH,chromosome:HSCHR3_1_CTG1"}
```

Reply:
Thanks. I've found a few other occurrences of this